geoportal/geoportailv3_geoportal/static-ngeo/js/search/SearchController.js

```diff
@@ -350,13 +355,13 @@ const exports = function($scope, $window, $compile,
     display: function(feature) {
       if (feature) {
         feature['dataset'] = 'backgroundLayers'
-        return this.gettextCatalog.getString(feature.get('label'))
+        return this.gettextCatalog.getString(feature.name)
```

Comment:
Why did this change?

Reply:
the v4 store provides "Layer" objects which have the name attribute, whereas the legacy code used full featured OL layers, where the "label" property was set at creation.
https://github.com/Geoportail-Luxembourg/geoportailv3/blob/master/geoportal/geoportailv3_geoportal/static-ngeo/js/GetWmtsLayerFactory.js#L105
https://github.com/Geoportail-Luxembourg/geoportailv3/blob/master/geoportal/geoportailv3_geoportal/static-ngeo/js/GetWmsLayerFactory.js#L57
https://github.com/Geoportail-Luxembourg/geoportailv3/blob/master/geoportal/geoportailv3_geoportal/static-ngeo/js/Themes.js#L159